pkg/console/subresource/consoleserver/config_builder_test.go

```diff
@@ -610,6 +610,61 @@ customization:
     - id: notagsorsubcategory
       label: No tags or subcategory
 providers: {}
+`,
+		},
+		{
+			name: "Config builder should handle custom add page with empty list as disabled actions",
```

Comment:
Not sure it makes sense to test empty list, since thats done by other tests are well.

Reply:
Removed this test